### DISCUSSION TITLE
DNS and/or IP addresses of AlmaLinux repositories

### DISCUSSION THREAD

**Discussion by aqua-calc** [AlmaLinux/almalinux-deploy#63] [Q&A]
Hello,

Could you please publish the DNS names and/or IP addresses of AlmaLinux repositories, so we can adjust our firewall to allow for CentOS to AlmaLinux migration?

Thank you!

P.S. At present. we're getting this error:

AlmaLinux 8 - BaseOS                            9.2 kB/s | 1.5 kB     00:00
Error: Failed to download metadata for repo 'baseos': repomd.xml parser error: Element <repomd> was not found - Bad repomd file
Run dnf distro-sync -y. Exit code: 1                                  ERROR

**Comment by andrewlukoshko** [comment]
Hi!
Mirrors DNS names can be found here: https://mirrors.almalinux.org/
But it will be hard to maintain your firewall rules because new mirrors are added almost every day.
